docs/synthetics/ci-cd/ci-cd-example-simple.md

```diff
@@ -0,0 +1,99 @@
+title: Example - Simple Deployment Setup
+description: A showcase of how to set up the CI/CD integration when our deployment setup is handled within GitHub and contains contextual information needed to link the deployment event to a specific commit
+
+If your setup is such that you create deployments within GitHub (like through automatic deployment tools such as Vercel), you can make the workflow for this action trigger on the `deployment_status` event and simply configure the variables needed for the action. Because of the simplicity of such a setup, the only required permissions are those that allow the workflow to read the deployment event and extract the URL of the environment that's being tested.
+
+
+## Workflow example
+
+Save the following file as `.github/workflows/sematext_synthetics_check.yaml`. Make sure to push it to your **main branch**, as that's where GitHub fetches workflows from when it's about to execute them.
+
+Remember that you should edit the variables which are sent to the action as per the instructions provided in the comments next to them. 
+
+```
+MONITOR_GROUP_ID: <your_monitor_group_id>               # Replace with your actual [Monitor Group ID](/docs/synthetics/ci-cd/ci-cd-installation#the-sematext-cicd-github-action)
+REGION: <region>                                        # Replace with your Sematext Cloud Region ('EU' or 'US')
+```
+
+The first step is to extract the URLs. If you're using Vercel, this is already handled for you, so no further action is needed.  If you're not using Vercel, then simply extract the URL you want to run the tests for (the `DEPLOYMENT_URL` variable) in a way compatible with how your setup is configured. If there's only one URL that you plan on monitoring, then you can turn the **Dynamic URL** option off for your monitors and avoid passing any `TARGET_URL` to the action.
```

Comment:
@HSSalman I feel like we need to explain this a bit but I don't know how.

Reply:
Partially addressed in the latest changes on the main branch